### ISSUE TITLE
Merge ttcli and Spring Initializr

### ISSUE THREAD

**Issue by tvogel8570** [wimdeblauwe/ttcli#45]
I'm trying to integrate your book section 4.2.2.2 with an existing project that was created on Spring Initializr.  It was quite tedious to merge the two.  Not sure the best way to make this easier, perhaps a set of guidelines about what is created, what needs to be moved and what needs to be merged would be of assistance.

Nirvana would be the ttcli tool can be run in a directory with an existing Spring Boot project and "it just works"!

**Comment by wimdeblauwe** [comment]
I agree that having the ttcli tool work on an existing project would be very nice, but I believe it would be quite hard to achieve given the many possible states the project might be in.

**Comment by garyhellman** [comment]
Hi Wim 
Thank you for your books and ttcli
I found that I needed to add a proxy to be able to use ttcli in my environment
Currently, I have the proxy hardcoded in the springBootInitializrClient Bean (and hardcoded "isReachable" to be true)

I suppose I will work towards getting it from environment variables
Please give suggestions and/or comments

Thanks!
